das/metrics.go

```diff
@@ -0,0 +1,132 @@
+package das
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric/global"
+	"go.opentelemetry.io/otel/metric/instrument"
+	"go.opentelemetry.io/otel/metric/instrument/asyncint64"
+	"go.opentelemetry.io/otel/metric/instrument/syncfloat64"
+	"go.opentelemetry.io/otel/metric/instrument/syncint64"
+
+	"github.com/celestiaorg/celestia-node/header"
+)
+
+var (
+	meter = global.MeterProvider().Meter("das")
+)
+
+type metrics struct {
+	sampled    syncint64.Counter
+	sampleTime syncfloat64.Histogram
+
+	newHead syncint64.Counter
+
+	busyWorkers        asyncint64.Gauge
+	isRunning          asyncint64.Gauge
```

Comment:
I'm wondering if we should keep `isRunning`- would it be clear from the other stats?

Reply:
you are right, any async metric submission is essentially "Hey, I'm running!" msg. Removed redundant metric, thank you!